tests/typed-holes/Example1.hs

```diff
@@ -1,6 +1,6 @@
 {-@ LIQUID "--expect-error-containing=Hole Found" @-}
 {-@ LIQUID "--exact-data-cons" @-}
-{-@ LIQUID "--allow-typed-holes" @-}
+{-@ LIQUID "--warn-on-term-holes" @-}
```

Comment:
This could be added to the `basic-pos` testsuite. For `scripts/test/test_plugin.sh` to pick up the module, it will need to be listed in `tests/tests.cabal`.

Reply:
I just found that it is actually listed in `tests.cabal`. A separate executable seems overkill for a single test. It would be terser to put it in an existing executable.

Reply:
I added more tests to this. I was thinking of adding all my examples to it. But maybe it is a overkill